pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "project-evergreen"
+version = "0.1.0"
+description = "Project for CSE312 in Fall 2024."
+authors = ["Eric-Butcher <ebutcher66@gmail.com>"]
```

Comment:
I think this is missing a few entries. Otherwise lgtm

Reply:
will revise. I admit I just copied these files from Eric.

Reply:
I added everyone as authors using school email addresses.